app/enterprise/changelog.md

```diff
@@ -4,6 +4,154 @@ no_search: true
+
+#### Core
+This release includes the addition of a new schema entity validator: `mutually_exclusive`. Before, the
+`only_one_of` validator required at least one of the fields included be configured. This new entity validator allows
+only one or neither of the fields be configured.
+[#7765](https://github.com/Kong/kong/pull/7765)
+
+#### Configuration
+- Enable IPV6 on `dns_order` as unsupported experimental feature.
+  [#7819](https://github.com/Kong/kong/pull/7819).
+- The template renderer can now use `os.getenv`.
+  [#6872](https://github.com/Kong/kong/pull/6872).
+
+#### Hybrid Mode
+- Data plane is able to eliminate some unknown fields when Control Plane is using a more modern version.
+  [#7827](https://github.com/Kong/kong/pull/7827).
+
+#### Admin API
+- Add support for HEAD requests.
+  [#7796](https://github.com/Kong/kong/pull/7796)
+- Improve support for OPTIONS requests.
+  [#7830](https://github.com/Kong/kong/pull/7830)
+
+#### Plugins
+- **New plugin:** [jq](/hub/kong-inc/jq) (`jq`)
+  The jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+- [Kafka Log](/hub/kong-inc/kafka-log) (`kafka-log`)
+  The Kafka Log plugin now supports TLS, mTLS, and SASL auth.
+- [Kafka Upstream](/hub/kong-inc/kafka-upstream) (`kafka-upstream`)
+  The Kafka Upstream plugin now supports TLS, mTLS, and SASL auth.
+- [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced) (`rate-limiting-advanced`)
+  - The Rate Limiting Advanced plugin now has a new identifier type: `path`, which allows rate limiting by matching request paths.
+  - The plugin now has a `local` strategy in the schema. The local strategy automatically sets `config.sync_rate` to -1.
+- [OPA](/hub/kong-inc/opa) (`opa`)
+  The OPA plugin now has a request path parameter, which makes setting policies on a path easier for administrators.
+- [Canary](/hub/kong-inc/canary) (`canary`)
+  The Canary plugin now has the option to hash on the header (falls back on IP, and then random).
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  Upgrade to v2.0.x to maintain version compatibility with older data planes.
+  - The OpenID Connect plugin can now handle JWT responses from a `userinfo` endpoint.
+  - The plugin now supports JWE Introspection.
+- [AWS-Lambda](/hub/kong-inc/aws-lambda) (`aws-lambda`) 
+  The plugin will now try to detect the AWS region by using `AWS_REGION` and
+  `AWS_DEFAULT_REGION` environment variables (when not specified with the plugin configuration).
+  [#7765](https://github.com/Kong/kong/pull/7765)
+- [Datadog](/hub/kong-inc/datadog) (`datadog`)
+  The Datadog plugin now allows `host` and `port` config options be configured from `ENV` variables.
+  [#7463](https://github.com/Kong/kong/pull/7463)
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) 
+  The new `data_plane_cluster_cert_expiry_timestamp` exposes DP cert expiry timestamp.
+  [#7800](https://github.com/Kong/kong/pull/7800).
```

Comment:
- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`)
    A new metric `data_plane_cluster_cert_expiry_timestamp` is added to expose the Data Plane's cluster_cert expiry 
    timestamp for improved monitoring in Hybrid Mode. [#7800](https://github.com/Kong/kong/pull/7800).